packages/model-viewer/src/features/controls.ts

```diff
@@ -366,12 +366,20 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
     }
 
     // Provided so user code does not have to parse these from attributes.
-    getMinimumFieldOfView(): number|undefined {
-      return this[$controls].options.minimumFieldOfView;
+    getMinimumFieldOfView(): number {
+      const value = this[$controls].options.minimumFieldOfView;
+      if(value === undefined) {
+        throw new Error('minimumFieldOfView was undefined');
```

Comment:
I understand the sentiment here, but we don't do this anywhere else in our code base. The fact is this is always defined, and if you're concerned, it's better to write a test for it.